src/actions/task_actions.js

```diff
@@ -0,0 +1,28 @@
+import {SAVE_TASK} from './type';
+import axios from 'axios';
+
+export const saveTask = evt => {
+    evt.preventDefault();
+    let form = evt.target;
+    let input = form.childNodes[0];
+    let url = 'http://localhost:8080/tasks';
+
+    var task = {
+        name: input.value,
+        createdAt: new Date(),
+        completed: false
+    };
+
+    return (dispatch) => {
+        return axios.post(url, task).then(response => {
+            dispatch({
+                type: SAVE_TASK,
+                payload: response.data
+            });
+
+            form.reset();
```

Comment:
I think it's better to put the code for the form (evt.preventDefault, form.reset etc) inside a handler method in the Task component, it keeps our actions cleaner.